src/groups/mqb/mqbs/mqbs_filestoreprotocol.h

```diff
@@ -998,7 +999,7 @@ struct RecordHeader {
     //   +---------------+---------------+---------------+---------------+
     //   |0|1|2|3|4|5|6|7|0|1|2|3|4|5|6|7|0|1|2|3|4|5|6|7|0|1|2|3|4|5|6|7|
     //   +---------------+---------------+---------------+---------------+
-    //   | Type  |      Flags            |  Sequence Number Upper Bits   |
+    //   |   Flags               |  Type |  Sequence Number Upper Bits   |
```

Comment:
Referencing [`mqbs::FileHeader`](https://github.com/bloomberg/blazingmq/blob/8702c6405ea765f9fb6a25f6db93cf80915899aa/src/groups/mqb/mqbs/mqbs_filestoreprotocol.h#L299), our ascii illustrations follow a big-endian convention, thus you should switch the order of `Flags` and `Type` back to what it was.